src/table.ts

```diff
@@ -778,133 +749,46 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       rowStream?.removeListener('end', originalEnd);
     };
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    userStream.end = (chunk?: any, encoding?: any, cb?: () => void) => {
-      rowStreamUnpipe(rowStream, userStream);
-      userCanceled = true;
-      if (activeRequestStream) {
-        activeRequestStream.abort();
-      }
-      if (retryTimer) {
-        clearTimeout(retryTimer);
-      }
-      return originalEnd(chunk, encoding, cb);
-    };
-
-    const makeNewRequest = () => {
-      // Avoid cancelling an expired timer if user
-      // cancelled the stream in the middle of a retry
-      retryTimer = null;
-
-      const lastRowKey = chunkTransformer ? chunkTransformer.lastRowKey : '';
+    (() => {
```

Comment:
I did this so that the PR would maintain the indent making it easier to review. It won't affect how the code works and we can remove it later.